docs/code/MlNetCookBook.md

```diff
@@ -578,6 +578,48 @@ var biases = modelParameters.GetBiases();
 
 ```
 
+## How do I look at the global feature importance?
+The below snippet shows how to get a glimpse of the the feature importance, or how much each column of data impacts the performance of the model.
+
+```csharp
+var transformedData = model.Transform(data);
+
+var featureImportance = context.Regression.PermutationFeatureImportance(model.LastTransformer, transformedData);
+
+foreach (var metricsStatistics in featureImportance)
+{
+    Console.WriteLine($"Root Mean Squared - {metricsStatistics.Rms.Mean}");
```

Comment:
>Console.WriteLine($"Root Mean Squared - {metricsStatistics.Rms.Mean}"); [](start = 4, length = 71)

Explain a bit above about what this is calculating. It's not giving the RMS, but the difference in RMS for each feature if the feature were to be replaced with a random value.

Also, I would print "Feature I: Difference in RMS" rather than just the RMS.